gio2/lib/gio2.rb

```diff
@@ -52,4 +52,6 @@ class << self
       loader.load("Gio")
     end
   end
+
+  Version = GLib::Version
```

Comment:
This is just comment. You don't have to take care.
IMO, I want to put this `Version` constant into near another constant definition.